src/Tizen.BatteryMonitor/Tizen.BatteryMonitor/BatteryMonitor.cs

```diff
@@ -0,0 +1,159 @@
+﻿/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+
+namespace Tizen.BatteryMonitor
+{
+    static internal class Globals
+    {
+        internal const string LogTag = "Tizen.BatteryMonitor";
+    }
+
+    /// <summary>
+    /// Provides information related to the power consumption by applications or by hardware resources on a battery-powered device for a given duration of time.
+    /// </summary>
+    /// <feature>http://tizen.org/feature/battery</feature>
+    /// <privilege>http://tizen.org/privilege/systemmonitor</privilege>
+    /// <since_tizen> 7 </since_tizen>
+    public static class BatteryMonitor
```

Comment:
How about rename to PowerUsageMonitor?
BatteryMonitor confuse with `Tizen.System.Battery`

Reply:
yes, considering the confusion with Tizen.System.Battery.
We will rename it to PowerUsage and add it in Tizen.System 
then the access will be Tizen.System.PowerUsage